reference/sql/statements/flashback-table.md

```diff
@@ -0,0 +1,84 @@
+---
+title: FLASHBACK TABLE
+summary: Learn how to recover tables using the `FLASHBACK TABLE` statement.
+category: reference
+---
+
+# FLASHBACK TABLE
+
+The `FLASHBACK TABLE` statement is introduced in TiDB 4.0. You can use the `FLASHBACK TABLE` statement to restore the tables and data dropped by `DROP` or `TRUNCATE` within the Garbage Collection (GC) lifetime.
+
+Use the following command to query the TiDB cluster's `tikv_gc_safe_point` and `tikv_gc_life_time`. As long as the table dropped by `DROP` or `TRUNCATE` is after the `tikv_gc_safe_point` time, you can restored the table using the `FLASHBACK TABLE` statement.
+
+{{< copyable "sql" >}}
+
+  ```sql
+  select * from mysql.tidb where variable_name in ('tikv_gc_safe_point','tikv_gc_life_time');
+  ```
+
+## Syntax
+
+{{< copyable "sql" >}}
+
+```sql
+FLASHBACK TABLE table_name [TO other_table_name]
+```
+
+## Notes
+
+If a table is dropped and the GC lifetime has passed, you can no longer use the `FLASHBACK TABLE` statement to recover the deleted data. Otherwise, an error will be returned, which is similar to `Can't find dropped / truncated table 't' in GC safe point 2020-03-16 16:34:52 +0800 CST`.
+
+Pay attention to the following conditions and requirements when you enable TiDB Binlog and use the `FLASHBACK TABLE` statement:
+
+* The downstream cluster must also support `FLASHBACK TABLE`.
+* The GC lifetime of the slave cluster must be longer than that of the master cluster.
+* The delay of replication between the upstream and downstream might also cause the failure to recover data downstream.
+* If an error occurs when TiDB Binlog is replicating a table, you need filter that table in TiDB Binlog and manually import all data of that table.
+
+## Example
+
+- Recover the table data dropped by `DROP`:
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    DROP TABLE t;
+    ```
+
+    {{< copyable "sql" >}}
+
+    ```sql
+    FLASHBACK TABLE t;
+    ```
+
+- Recover the table data dropped by `TRUNCATE`. Because the dropped table still exists, you need to rename the table to be recovered. Otherwise, an error will be returned indicating that table `t` already exists.
```

Comment:
```suggestion
- Recover the table data dropped by `TRUNCATE`. Because the truncated table still exists, you need to rename the table to be recovered. Otherwise, an error will be returned indicating that table `t` already exists.
```